apps/api/src/app/workflows-v2/usecases/validate-content/prepare-and-validate-content/prepare-and-validate-content.usecase.ts

```diff
@@ -292,12 +297,48 @@ export class PrepareAndValidateContentUsecase {
         controlValueToContentIssues,
         item,
         StepContentIssueEnum.MISSING_VALUE,
-        `${toSentenceCase(item)} is missing`,
+        `${toSentenceCase(item)} is required`,
         item
       );
     }
   }
 
+  private overloadMinValueIssues(
+    jsonSchema: JSONSchemaDto,
+    controlValues: Record<string, unknown>,
+    contronValueIssues: Record<string, ContentIssue[]>
+  ) {
+    if (typeof jsonSchema !== 'object') {
+      return;
+    }
+
+    let schemaToExtractDefaults: JSONSchemaDefinition = jsonSchema;
+    if (jsonSchema.anyOf && jsonSchema.anyOf.length > 0) {
+      schemaToExtractDefaults =
+        jsonSchema.anyOf.find((item) => {
+          return isMatchingJsonSchema(item, controlValues);
+        }) ?? jsonSchema[0];
+    }
+    const minValuesFromSchema = extractMinValuesFromSchema(schemaToExtractDefaults);
+
+    for (const [key, minValue] of Object.entries(minValuesFromSchema)) {
+      if (typeof controlValues[key] !== 'number') {
+        continue;
+      }
+      if (controlValues[key] >= minValue) {
+        continue;
+      }
+
+      this.addToIssues(
+        contronValueIssues,
+        key,
+        StepContentIssueEnum.MIN_VALUE,
+        `${toSentenceCase(key)} must be at least ${minValue}`,
+        key
+      );
+    }
```

Comment:
For each extracted field: if the control value is less than the min value, then add the issue.